docs/envvars.rst

```diff
@@ -134,7 +134,11 @@ applicable.
         in JSON format.
     * - XONSHRC
       - ``~/.xonshrc``
-      - Location of run control file.
+	``/etc/xonshrc`` (Linux and OSX) 
+	``%ALLUSERSPROFILE%\Application Data\xonsh\xonshrc`` (Windows)
```

Comment:
This default seems incorrect. This should be a list/tuple of both the system and user files. Right now they are out of order and not a list, just next to each other.